odoo/http.py

```diff
@@ -2177,18 +2180,15 @@ def dispatch(self, endpoint, args):
     def handle_error(self, exc: Exception) -> collections.abc.Callable:
         """
         Handle any exception that occurred while dispatching a request to
-        a `type='json'` route. Also handle exceptions that occurred when
+        a `type='odoo-rpc'` route. Also handle exceptions that occurred when
         no route matched the request path, that no fallback page could
         be delivered and that the request ``Content-Type`` was json.
 
         :param exc: the exception that occurred.
         :returns: a WSGI application
         """
         error = {
-            'code': 200,  # this code is the JSON-RPC level code, it is
-                          # distinct from the HTTP status code. This
-                          # code is ignored and the value 200 (while
-                          # misleading) is totally arbitrary.
+            'code': 0,  # we don't care of this code
```

Comment:
How about -32603 Internal error?
https://www.jsonrpc.org/specification#error_object

Reply:
nop, not going to do that,

using the actual code would say two things:
1. we send the precise jsonrpc code for what went wrong
2. we care about the code

we wouldn't be sending the correct jsonrpc code, e.g. for bad json, we don't care about that code

0 it is, it is wrong in all cases and it is pretty much saying "we don't care"